pkg/services/updatechecker/grafana.go

```diff
@@ -40,15 +55,15 @@ func (s *GrafanaService) IsDisabled() bool {
 }
 
 func (s *GrafanaService) Run(ctx context.Context) error {
-	s.checkForUpdates()
+	s.checkForUpdates(ctx)
 
 	ticker := time.NewTicker(time.Minute * 10)
 	run := true
 
 	for run {
 		select {
 		case <-ticker.C:
-			s.checkForUpdates()
+			s.checkForUpdates(ctx)
```

Comment:
Code might be simplified by returning an error here, checking it and log? Same applies for plugins update checker

Reply:
Nice catch, I have updated the code :+1: